cmd/koord-scheduler/app/config/config.go

```diff
@@ -30,6 +31,7 @@ type Config struct {
 	KoordinatorClient                koordinatorclientset.Interface
 	KoordinatorSharedInformerFactory koordinatorinformers.SharedInformerFactory
 	NRTSharedInformerFactory         nrtinformers.SharedInformerFactory
+	PodGroupInformerFactory          pginformers.SharedInformerFactory
```

Comment:
This implementation is not recommended anymore. In the past I tried a new implementation to pass CRD Client and Informer Factory that can be reused frequently, but I noticed that most of the CRD Client and Informer Factory are actually not reused. It is recommended to initialize the CRD Client and Informer Factory inside the plugin at first. Also start the Informer Factory inside the plugin and wait for the synchronization to complete.

Reply:
ok,solved